framework/DataObjects/PointSet.py

```diff
@@ -230,22 +260,14 @@ def _updateSpecializedOutputValue(self,name,value,options=None):
     value = np.atleast_1d(value).flatten()
     if len(value) > 1:
 
-      if options is None:
-        outputRow = -1
-        outputPivotVal = None
-        operator = None
-      else:
-        ## Not sure if any of these are necessary, but I am trying to replicate
-        ## the magic that takes place in the CsvLoader -- DPM 5/3/2017
-        outputRow = copy.deepcopy(options.get('outputRow',-1))
-        outputPivotVal = options.get('outputPivotValue',None)
-        operator = options.get('operator',None)
+      row = -1
+      if self._dataParameters is not None:
+        row = self._dataParameters.get('outputRow', -1)
+        operator = self._dataParameters.get('operator', None)
 
-      if outputRow is None:
-        if outputPivotVal is not None and 'end' in outputPivotVal:
-          outputRow = -1
-        # elif outputPivotVal != None:
-        #   outputPivotVal = float(outputPivotVal)
+      ## Zero or one indexed?
+      # if row  > 0:
+      #   row  -= 1
```

Comment:
Do you need these three commented lines?

Reply:
removed